# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../squashfs-tools/gcc49.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- squashfs4.3/squashfs-tools/mksquashfs.c.vanilla	2014-06-04 17:24:43.000000000 +0200
+++ squashfs4.3/squashfs-tools/mksquashfs.c	2014-06-04 17:33:22.000000000 +0200
@@ -1437,6 +1437,7 @@
 	cache_block_put(compressed_buffer);
 
 finished:
+;
 	pthread_cleanup_pop(0);
 
 	return buffer;
--- squashfs4.3/squashfs-tools/process_fragments.c.vanilla	2014-06-04 17:25:05.000000000 +0200
+++ squashfs4.3/squashfs-tools/process_fragments.c	2014-06-04 17:29:22.000000000 +0200
@@ -210,6 +210,7 @@
 	cache_block_put(compressed_buffer);
 
 finished:
+;
 	pthread_cleanup_pop(0);
 
 	return buffer;
